app/controllers/application_controller.rb

```diff
@@ -5,19 +5,17 @@ class ApplicationController < ActionController::Base
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery except: :service_sign_in_options
 
-  def current_tasklist_ab_test
-    GovukNavigationHelpers::CurrentTasklistAbTest.new(
-      current_tasklist: current_tasklist,
-      request: request
-    )
-  end
-  helper_method :current_tasklist_ab_test
-
 private
 
-  def current_tasklist
-    GovukNavigationHelpers::TasklistContent.current_tasklist(request.path)
+  def current_step_nav
```

Comment:
If we decide to keep using this we should introduce both this methods in the gem so all we have to do around all the frontend apps is to include the class. 
(Not something to do now)

Reply:
Yep, when we move to the published info, we can do a bit of a clean up again 👍